source/_integrations/sftp_client.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: SFTP Client
+description: Instructions on how to use  SFTP Client in Home Assistant.
+ha_category:
+  - Backup
+ha_iot_class: Local Polling
+ha_release: '2025.2'
+ha_config_flow: true
+ha_domain: sftp_client
+ha_codeowners:
+  - '@cyr-ius'
+ha_integration_type: service
+ha_quality_scale: platinum
+related:
+  - docs: /common-tasks/general/#backups
+    title: Creating backups in Home Assistant
+---
+
+This {% term integration %} allows you to perform Home Assistant backups to a server via SFTP. Once this integration is configured, your server will contain a new folder called backup where all backups will be stored. If you delete the folder, it will be automatically recreated as long as the {% term integration %} is enabled.
```

Comment:
```suggestion
The **SFTP Client** {% term integration %} allows you to perform Home Assistant backups to a server via <abbr title="secure file transfer protocol">SFTP</abbr>. Once this integration is configured, your server contains a new folder called backup where all backups will be stored. If you delete the folder, it will be automatically recreated as long as the {% term integration %} is enabled.
```